tests/tests/control_gfs_mpas

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+#
+# GFS test with MPAS dynamical core.
+#
+###############################################################################
+
+export TEST_DESCR="Build UWM with MPAS dynamical core for GFS application"
+
+export CNTL_DIR=control_gfs_mpas
+
+export LIST_FILES=""
```

Comment:
Why is 'LIST_FILES' empty? Are MPAS test not reproducible?

Reply:
There isn't any output yet.
I will add the MPAS logfile here. At least that will alert us if anyone broke what is working up until this point. @DusanJovic-NOAA What do you think?

Reply:
I'm not sure what kind of log files MPAS creates, but the files listed here are saved as baselines, so they must be reproducible from run to run. If log files contain any timing info or anything that might differ between runs, then that will not work.